src/components/top-bar/_top-bar.scss

```diff
@@ -28,6 +30,10 @@
     &:empty {
       margin: 0;
     }
+
+    .ad-notice {
```

Comment:
ad notice has its own css file. do it there

Reply:
@yairans it's not working when adding this to ad-notice.scss. the adding of pointer-events:auto to top-bar's children is stronger. we needed to add another level to make this one stronger.